contracts/game/src/game/interfaces.cairo

```diff
@@ -16,9 +17,10 @@ trait IGame<TContractState> {
         client_reward_address: ContractAddress,
         weapon: u8,
         name: felt252,
```

Comment:
@starknetdev note the following two changes to `new_game()`
1. golden_token_id changed from a u256 to a u8
2. added launch_tournament_winner_token_id: u32

The contract will refuse any/all launch_tournament_winner_token_ids until after the tournament is finished and settled, which happens via someone calling `settle_launch_tournament` once the tournament is complete. When this happens, the contract will set an internal launch_tournament_champion_collection which it will then use to process the launch_tournament_winner_token_id, providing them with one game per month, forever